### DISCUSSION TITLE
Minimum required memory to run?

### DISCUSSION THREAD

**Discussion by Jarrodsz** [docker-mailserver/docker-mailserver#3188] [Q&A]
I could not find any info in the docs as to what would be the minimum required amount of memory required to run this project. Aprox.

I know the anti virus feature uses a lot of memory but I won't be needing that. Anti spam filtering would be a nice feature. I plan to configure using IMAP and around 5 mailboxes to begin with. No fancy stuff beside the basics. 

What amount of memory would I have to aprox think about I will end up with?

**Comment by ontheair81** [comment]
Right now on my server: Docker mailserver container alone uses about 350 MB of RAM. Antivirus disabled.
Multiple accounts on multiple domains. At the moment there are two Thunderbird clients connected to the server.

You may want some additional containers (for retreiving certificates, providing autoconfig informations, etc.). A server with 4GB is more than sufficient, maybe even 2GB will work - depending on your other needs, choosen operating system, etc.

**Comment by Jarrodsz** [comment]
Ah great, I guess it could also use a swap partition if you configure it expliciet. Since mail server does not need a high performance.
